packages/react-app/src/hooks/useBounties.ts

```diff
@@ -0,0 +1,53 @@
+import { BountyCollection } from '../models/Bounty';
+import useSWR from 'swr';
+import axios from '../utils/AxiosUtils';
+
+// axios returns an AxiosResponse, with the payload in the `data` object
+const axiosFetcher = (url: string) => axios.get(url).then(({ data }) => data.data);
+
+interface BountySWRResponse {
+  isLoading: boolean;
+  isError: boolean;
+}
+
+interface useBountiesResponse extends BountySWRResponse {
+  bounties?: BountyCollection[]
+}
+
+interface useBountyResponse extends BountySWRResponse {
+  bounty?: BountyCollection
+}
+
+export function useBounties(url: string): useBountiesResponse {
+	/**
+   * Wraps the SWR hook with additional loading state for spinners
+   */
+	const { data, error } = useSWR<BountyCollection[], unknown>(url, axiosFetcher);
+	return {
+		bounties: data,
+		isLoading: !error && !data,
+		isError: !!error,
```

Comment:
`!!error` vs `error` ?

Reply:
!!error coalesces the error value as a boolean, it is equivalent to Boolean(error)